src/ekstat.erl

```diff
@@ -1,14 +1,20 @@
 -module(ekstat).
 
 %% API
--export([open/0, update/1, clear/1, read/1, read/2, read/3, read/4, read/5, read/6]).
+-export([open/0, close/1, clear/1, update/1, read/1, read/2, read/3, read/4, read/5, read/6]).
 
 -on_load(init/0).
 
+-include("async_nif.hrl").
+
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+-endif.
+
 %% Types
 -type kstat_handle() :: binary().
 -type kstat_pattern() :: '_' | integer() | string().
--type kstat_error() :: {error, {kstat, string()}}.
+-type kstat_error() :: {error, atom()} | {error, {atom(), string()}}.
 -type kstat_statistic() :: {Statistic::string(), Value::integer()} | {Statistic::string(), Value::string()}.
 -type kstat_row() :: {Class::string(), Module::string(), Instance::integer(), Name::string(), [kstat_statistic()]}.
```

Comment:
Line 19 is too long: -type kstat_row() :: {Class::string(), Module::string(), Instance::integer(), Name::string(), [kstat_statistic()]}..